diff --git NuttX/nuttx/include/cxx/cwchar NuttX/nuttx/include/cxx/cwchar
index 167f598..4b60226 100755
--- NuttX/nuttx/include/cxx/cwchar
+++ NuttX/nuttx/include/cxx/cwchar
@@ -54,6 +54,7 @@ namespace std
   using ::wctype_t;
   using ::size_t;
 
+#ifdef CONFIG_LIBC_WCHAR
   using ::btowc;
   using ::fwprintf;
   using ::fwscanf;
@@ -132,6 +133,7 @@ namespace std
   using ::wmemset;
   using ::wprintf;
   using ::wscanf;
+#endif
 };
 
 #endif // __INCLUDE_CXX_CWCHAR
